clippy_lints/src/use_self.rs

```diff
@@ -73,12 +76,10 @@ impl<'a, 'tcx> Visitor<'tcx> for UseSelfVisitor<'a, 'tcx> {
         if self.item_path.def == path.def &&
            path.segments
             .last()
-            .expect("segments should be composed of at least 1 element")
+            .unwrap()
```

Comment:
I added another expect() call with the same message. I thought the message doesn't provide additional debug information, it repeats what the code is trying. So I removed them. Should I put them back?